README.md

```diff
@@ -1,10 +1,25 @@
 # beebop
 
-Start the Express server with 
+Clone the repository to your computer with
 ```
-node index.js
+git clone git@github.com:bacpop/beebop.git
 ```
-Then go to `client/` and start the vue App:
+
+To start the Express server, go into the `app/server/` directory and run
+```
+npm install
+npm run express
```

Comment:
this fails for me:

```
$ npm run express

> beebop@1.0.0 express
> ts-node src/index.ts

sh: 1: ts-node: not found
```

Do we rely on a global ts-node installation? Or (probably better) should this be part of the package.json?